docker/docker-compose/docker-compose.yml

```diff
@@ -8,12 +8,12 @@ x-definitions: &env
     # IMPORTANT!!
     # Change this to support both local and remote plugins
     # Should be resolvable from inside and outside the docker network
-    BG_MQ_HOST: localhost
+    BG_MQ_HOST: rabbitmq
```

Comment:
I know this is annoying, but we have to keep this as `localhost`. Changing this to `rabbitmq` means that local plugins will work "out of the box", but remote plugins will break. On the other hand, having it be `localhost` means that remote plugins will work but local plugin will break.

There's no great way to have both work out of the box, unfortunately. And since our docs / tutorials are written for remote plugins it's better to optimize for that case.